conf/openshift/openshift.json

```diff
@@ -222,15 +227,17 @@
       }
     },
     {
-      "kind": "DeploymentConfig",
-      "apiVersion": "v1",
+      "kind": "StatefulSet",
+      "apiVersion": "apps/v1beta1",
```

Comment:
Does `v1beta1` require a beta version of minishift? Or can I just use the most recent regular release of minishift?

Reply:
Regular release is fine. I was running on v1.14

Reply:
Ok, great. Thanks.